packages/node-bridge/helpers.ts

```diff
@@ -309,3 +310,7 @@ export function createServerWithHelpers(
 
   return server;
 }
+
+function colorRed(content: string) {
+  return `\x1b[31m${content}\x1b[0m`;
```

Comment:
Non-blocking: Can `node-bridge` pull in `chalk`? It should be free assuming `ncc` bundles it, right?

Reply:
I don't know the full impact of adding dependencies there. Maybe someone else can speak to it. I'd be happy to pull in chalk here.

Reply:
I guess another benefit of using `chalk` is the whole `--no-colors` thing, ya?

Reply:
No we should not add dependencies because any increase in size could put users over the lambda limit of 50MB